.github/workflows/ruff.yml

```diff
@@ -0,0 +1,20 @@
+name: Ruff check
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: chartboost/ruff-action@v1
+        with:
+          version: 0.0.261
+          line-length: "150"
```

Comment:
Do we really need to specify all the Ruff parameters in both workflow and pyproject? For me it is better to have all the configurations in pyproject only!

Reply:
Sorry missed that. Moved the configs to pyproject.